index.js

```diff
@@ -149,6 +149,8 @@ class RNCallKeep {
 
   sendDTMF = (uuid, key) => RNCallKeepModule.sendDTMF(uuid, key);
 
+  toggleAudioRouteSpeaker = (uuid, useSpeaker) => RNCallKeepModule.toggleAudioRouteSpeaker(uuid, useSpeaker);
```

Comment:
Can you check to only call it on Android please ?

Reply:
yup, let me make this Android-specific function